### DISCUSSION TITLE
Default People Search Page Display

### DISCUSSION THREAD

**Discussion by bridgestateu** [microsoft-search/pnp-modern-search#2930] [Q&A]
Is there a way to display a complete list of refiners as a default page configuration without displaying search results? I would like users to see a complete list of Division names, Department names, Job Titles, and Last Names and have them selectable to produce search results. 

![image](https://user-images.githubusercontent.com/77447096/234296155-6b6b174b-7ed1-4b90-9801-4dd21824605e.png)

**Comment by kasperbolarsen** [comment]
Which is exactly what you get by using the default setup, if the default search is set to use * , except that the end users will see the first 50 hits.
Any particular reason why this is not an acceptable option?
